### ISSUE TITLE
show which attributes contribute to score

### ISSUE THREAD

**Issue by ckingdon** [CSTARS/ecosis-data-tool#8]
When I load my metadata files I see that 1 of my 15 metadata fields matches. As I add new metadata my score increases.

It would be great if I could see which of the 15 attributes have been counted.

**Comment by jrmerz** [comment]
I think I need a little more info for this request.  You are seeing your score increase in the main table and would like to know which fields helped your score from the table?

**Comment by ckingdon** [comment]
Yes. More accurately, I'd like to know which fields I've already completed.

**Comment by jrmerz** [comment]
Ok, can look to add a popover for the label when it is clicked.
